manifests/u/Ubisoft/Connect/151.2.11050/Ubisoft.Connect.installer.yaml

```diff
@@ -1,18 +1,18 @@
-# Automatically updated by the winget bot at 2024/Mar/28
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
-
-PackageIdentifier: Ubisoft.Connect
-PackageVersion: 151.1.0.11048
-InstallerType: nullsoft
-Scope: machine
-InstallModes:
-- interactive
-- silent
-UpgradeBehavior: install
-ProductCode: Uplay
-Installers:
-- Architecture: x86
-  InstallerUrl: https://static3.cdn.ubi.com/orbit/launcher_installer/UbisoftConnectInstaller.exe
-  InstallerSha256: 48AA6A6FC3A3C255F38774A14B20D5C9FDA156EF5435A31A45B44DFF50CDC3E0
-ManifestType: installer
-ManifestVersion: 1.5.0
+# Created with YamlCreate.ps1 v2.4.1 $debug=AUSU.CRLF.5-1-26100-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Ubisoft.Connect
+PackageVersion: 151.2.11050
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://static3.cdn.ubi.com/orbit/launcher_installer/UbisoftConnectInstaller.exe
+  InstallerSha256: 34478FB74672E95EC16BE1826901F85581410CC89BF24DCF09E6B8BF82E3D6BF
```

Comment:
```suggestion
  InstallerSha256: 48AA6A6FC3A3C255F38774A14B20D5C9FDA156EF5435A31A45B44DFF50CDC3E0
```

Reply:
This is the hash for the old version. When I test in a local sandbox, I get the hash matching `D6BF`. 

![image](https://github.com/microsoft/winget-pkgs/assets/12611259/1b9c2593-fa71-411d-8745-74f9c2a0a00f)

Reply:
I'm not sure of the path forward from here.

Reply:
> I'm not sure of the path forward from here.

What happens when you try to validate in your VM? Do you get a hash mismatch there?

Reply:
This package installs and launches normally in a WIndows 10 VM. Unfortunately, this isn't a label that I can waiver or get around.

Reply:
@denelon - Thoughts?

Reply:
Is this an issue where we're hitting different endpoints and getting different hash values?

Reply:
I don’t think so, since its pointing to a single static endpoint, but the Pipelines are always getting a different hash (from the previous version)